testing&Evaluation.ipynb

```diff
@@ -0,0 +1,267 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
```

Comment:
<img width="705" alt="image" src="https://github.com/geo-smart/swe-workflow-book/assets/45752727/c945f24c-cd3e-4fb6-ba3a-be24f1faeb0c">

Can you run the code and commit the notebook with the output log?